demo/demo.md

```diff
@@ -1,233 +1,312 @@
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/warning.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/warning.html -->
   <auro-alert type="warning">Warning. Session timed out. Look for a confirmation email to verify your transaction.</auro-alert>
-</div>
-
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
 <auro-accordion alignRight>
-  <span slot="trigger">See code</span>
-
-  ```html
-  <auro-alert type="warning">Warning. Session timed out. Look for a confirmation email to verify your transaction.</auro-alert>
-  ```
-</auro-accordion>
-
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/warning.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/warning.html -->
+
+```html
+<auro-alert type="warning">Warning. Session timed out. Look for a confirmation email to verify your transaction.</auro-alert>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
 ### Information
 
 <div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/information.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/information.html -->
   <auro-alert type="information">You are confirmed on Flight 20 to Aruba.</auro-alert>
-</div>
-
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
 <auro-accordion alignRight>
-  <span slot="trigger">See code</span>
-
-  ```html
-  <auro-alert type="information">You are confirmed on Flight 20 to Aruba.</auro-alert>
-  ```
-</auro-accordion>
-
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/information.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/information.html -->
+
+```html
+<auro-alert type="information">You are confirmed on Flight 20 to Aruba.</auro-alert>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
 ### Success
 
 <div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/success.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/success.html -->
   <auro-alert type="success">Your status with flight 20 to Aruba had been updated.</auro-alert>
-</div>
-
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
 <auro-accordion alignRight>
-  <span slot="trigger">See code</span>
-
-  ```html
-  <auro-alert type="success">Your status with flight 20 to Aruba had been updated.</auro-alert>
-  ```
-</auro-accordion>
-
-## Multiline alerts
-
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/success.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/success.html -->
+
+```html
+<auro-alert type="success">Your status with flight 20 to Aruba had been updated.</auro-alert>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+## Multiline alerts

```

Comment:
I would remove "Multiline alerts", no icon, and a11y alerts from the demo.md page and move that over to api.md if it does not already exist.

Reply:
sure.

Reply:
oh, also the a11y in the api.md would be explained per property name (`hidden`, `hiddenVisually`, and `hiddenAudible`) instead of a story like in demo.md